src/TDB2.cpp

```diff
@@ -51,7 +52,11 @@ static void dependency_scan(std::vector<Task>&);
 
 ////////////////////////////////////////////////////////////////////////////////
 void TDB2::open_replica(const std::string& location, bool create_if_missing) {
-  _replica = tc::new_replica_on_disk(location, create_if_missing);
+  try {
+    _replica = tc::new_replica_on_disk(location, create_if_missing);
+  } catch (rust::Error& err) {
+    throw format("Could not open replica at {1}: {2}", location, err.what());
```

Comment:
Would it make sense to modify https://github.com/GothenburgBitFactory/taskwarrior/blob/0b286460b69042b1ed409b981e42ea34c6172574/src/taskchampion-cpp/src/lib.rs#L468 instead so all callers get this extra context in the error message?

Reply:
Yeah, good point!

Reply:
In fact, it should be fixed in TaskChampion..